assets/src/frontend/components/FeeKeypad.vue

```diff
@@ -78,16 +78,15 @@ export default {
             this.displayValue='';
         },
         change(value){
-            if ( !isNaN(value) ) {
-                this.displayValue = value;
-                this.input = this.displayValue;
-            } else {
-                this.input = this.displayValue;
-                if ( this.displayValue == '' ) {
-                    this.input = '';
-                }
+            let displayValue  = value.replace( ".", wepos.currency_format_decimal_sep );
+            let originalValue = value.replace( wepos.currency_format_decimal_sep, "." );
```

Comment:
Please always, make the one time use variable data type to `const`.
And similar is applicable for `displayValue`
Also everywhere is using single quote, please convert that `""` to that.